src/main/java/com/jcabi/github/RtReleases.java

```diff
@@ -138,4 +172,17 @@ public void remove(final int number) throws IOException {
             .assertStatus(HttpURLConnection.HTTP_NO_CONTENT);
     }
 
+    /**
+     * Default URI from releases.
+     * @param repo Repository
+     * @return Release URI.
```

Comment:
@weisebrazil remove dot's after the `@param`/`@return` description (dot's should be only in the method/constructor description)